### PR TITLE
Introduce a Liquid tag to display code samples with different languages.

### PR DESCRIPTION
**Pull request by matthew-carroll**
Introduce a Liquid tag to display code samples with different languages.

### CODE REVIEW

File: src/_assets/js/main.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ $(function () {
   setupClipboardJS();
 
   // New (dash) tabs
-  setupTabs($('.nav-tabs'), 'io.flutter.tool-id');
```
Comment (xster) [inline src/_assets/js/main.js]: #editor-setup was here previously right? Is that using the new tag too? If not, are we breaking it?

Comment (matthew-carroll) [reply src/_assets/js/main.js]: Where is the #editor-setup code example?

Comment (xster) [reply src/_assets/js/main.js]: https://github.com/flutter/website/blob/master/src/_assets/js/main.js#L25

**Comment by xster** [comment]
Can you push this to your staging branch as well?

**Comment by xster** [comment]
Doh, I can't change the merge base from the UI here directly. Can you do an interactive rebase of 356c448 and cc36363 onto master directly? It should merge cleanly since there aren't too many files.

**Comment by matthew-carroll** [comment]
@xster I added back the editor configuration in main.js. I also converted the iOS samples to use the tag. I adjusted the behavior of the tag after seeing some different practices in the iOS code.

Can you take another look and let me know what you think?

**Comment by matthew-carroll** [comment]
The CLA warning was the result of the rebasing process. After changing the target branch, the only commits are my own, so I manually added the CLA tag.

File: src/_plugins/code_tabs.rb [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+#   // Swift code here
+#   ```
+#   {% endsamplecode %}
+#
+# The "samplecode" tag accepts a single parameter that represents the title of this sample. The
+# title is not displayed on the page, but it is injected into a number of HTML attributes to
+# differentiate itself from other code samples. The "samplecode" tag is a block tag and therefore
+# must be closed with a "endsamplecode" tag.
+#
+# The "sample" tag accepts one parameter: the name of the language as it should appear to
```
Comment (xster) [inline src/_plugins/code_tabs.rb]: These languages appear visually as tabs

**Comment by sfshaza2** [comment]
I see @xster has approved. Merging.

**Pull request merged**